### DISCUSSION TITLE
Item hover animation

### DISCUSSION THREAD

**Discussion by willuhmjs** [deep-sea-tactics/website#20] [Q&A] [answered]
Currently, Item components animate like buttons when hovered over, which may not be the best animation. Should the animation be replaced by a different animation or removed?

**Comment by tristan-f-r** [comment, marked as answer]
For the item components, if they're going to hold only dates (and not clickable items) the hover effect should be removed.

**Comment by dylansison** [comment]
if it’s the donate things, as you drag across it should like grow a little bit to pull attention and distract adhd kids and adults
